src/stream.c

```diff
@@ -1083,7 +1083,7 @@ int read_dmx(sockets *s)
 	if (!send)
 		return 0;
 
-	ad->flush = 0;
+	ad->flush = flush_all;
```

Comment:
I think flush_all is not needed at all by looking at the code.

Reply:
Ok. So perhaps it's a good idea to remove all code with `flush_all`.

Reply:
Yes is better